runtime/cudaq/platform/default/rest_server/helpers/RestRemoteServer.cpp

```diff
@@ -97,6 +97,14 @@ class RemoteRestRuntimeServer : public cudaq::RemoteRuntimeServer {
   // Server to exit after each job request.
   // Note: this doesn't apply to ping ("/") endpoint.
   bool exitAfterJob = false;
+  // Time-point data
+  std::optional<std::chrono::time_point<std::chrono::system_clock>>
+      requestStart;
+  std::optional<std::chrono::time_point<std::chrono::system_clock>>
+      simulationStart;
+  std::optional<std::chrono::time_point<std::chrono::system_clock>>
+      simulationEnd;
```

Comment:
Nit: I would recommend using `high_resolution_clock` instead of `system_clock` because the system clock can be updated with NTP (or any other system time updates) whereas `high_resolution_clock` is better suited for these "elapsed time" situations. It may not be worth changing here, but I wanted to bring it up for any potential future timing code.

Reply:
Agree. Fixed as recommended.